client/src/modules/list/components/ListHeader.jsx

```diff
@@ -165,11 +178,13 @@ class ListHeader extends PureComponent {
       />
     ) : (
       <Fragment>
-        {description ? (
+        {description || !isOwner ? (
```

Comment:
remove ternay here. 
leave just
```
if(description){
display descripiotn} 
```

and down below leave 
```
if(isOwner) {
render button - if he is not the owner, he will not see the button
}


```

it's gonna be simpler to understand

Reply:
I could to that, but then an additional check is needed. You have to check:
```
{description && (
 code
)}
{(isOwner  && !description && (
 code
)}
```
Because when you would be owner the button will always be displayed. Do you think we should check the description twice?

Reply:
put this version, it easier to understand
```

{description && (
 code
)}
{(isOwner  && !description && (
 code
)}
```